### PR TITLE
Create a new subsystem and constants data for PID on the solver

### PR DESCRIPTION
**Pull request by KenwoodFox**
Unfinished don't merge until ready

### CODE REVIEW

File: src/main/java/frc/robot/Constants.java [hunk 1]
```diff
@@ -159,9 +161,51 @@
...
+            public static double blue = 0.0;
+        }
+
+        public static final class Green{
+            public static double red = 0.0;
+            public static double green = 0.0;
+            public static double blue = 0.0;
+        }
+    }
+
```
Comment (boinary) [inline src/main/java/frc/robot/Constants.java]: For my understanding @KenwoodFox ~~

How come a lot of this is nested? 

This part of the program looks fantastic other than some tuning!

File: src/main/java/frc/robot/subsystems/Solver.java [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+public class Solver extends SubsystemBase {
+  // Motors
+  private static final TalonSRX solverMotor = new TalonSRX(Constants.CANIds.TalonSRX_Solver_Address); // Creates a new motor
+
+  /**
+   * Creates a new Solver.
+   */
+  public Solver() {
+    solverMotor.configFactoryDefault(); // Clear the programming to prevent weirdness
```
Comment (boinary) [inline src/main/java/frc/robot/subsystems/Solver.java]: what is weirdness?

File: vendordeps/REVColorSensorV3.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+            "binaryPlatforms": [
+                "windowsx86-64",
+                "windowsx86",
+                "linuxx86-64",
+                "linuxathena",
+                "linuxraspbian"
+            ]
+        }
+    ]
+}
```
Comment (boinary) [inline vendordeps/REVColorSensorV3.json]: EOL

File: src/main/java/frc/robot/Constants.java (hunk 1 above)
Comment (KenwoodFox) [reply src/main/java/frc/robot/Constants.java]: Yellow, Red and Blue are subclasses of the class color
this allows you to say Colors.Blue, Colors.Red and Colors.Green, allowing better readablity

File: src/main/java/frc/robot/subsystems/Solver.java (hunk 2 above)
Comment (KenwoodFox) [reply src/main/java/frc/robot/subsystems/Solver.java]: Motor controlers often tend to "glitch" upon sudden power shutdowns (power cycles) using the configFactoryDefault() protects against these.

File: vendordeps/REVColorSensorV3.json (hunk 3 above)
Comment (KenwoodFox) [reply vendordeps/REVColorSensorV3.json]: Not our problem, @revrobotics

File: src/main/java/frc/robot/subsystems/Solver.java (hunk 2 above)
Comment (boinary) [reply src/main/java/frc/robot/subsystems/Solver.java]: Have you tried turning it off and on again

Comment (KenwoodFox) [reply src/main/java/frc/robot/subsystems/Solver.java]: Yes thats why we use this line

Comment (boinary) [reply src/main/java/frc/robot/subsystems/Solver.java]: \:D

**Pull request merged**